src/guide/ssr/routing.md

```diff
@@ -0,0 +1,121 @@
+# Routing and Code-Splitting
```

Comment:
As an SSR app developer I am also curious how Critical CSS should be implemeneted correctly with this approach.

Reply:
I agree! However, this guide is not full and there are a few missing parts here. Here is an issue with 'what's currently missing': https://github.com/vuejs/docs-next/issues/971